internal/static/i18n/zh.yaml

```diff
@@ -0,0 +1,900 @@
+      UsernamePasswordNotAllowed: 不允许使用用户名/密码登录
+      MFA:
+        AlreadyExists: 多因素身份认证已经存在
+        NotExisting: 多因素身份认证不存在
+        Unspecified: 多因素身份认证无效
+    MailTemplate:
+      NotFound: 未找到默认邮件模板
+      NotChanged: 默认邮件模板未更改
+      AlreadyExists: 默认邮件模板已存在
+      Invalid: 默认邮件模板无效
+    CustomMessageText:
+      NotFound: 未找到默认消息文本
+      NotChanged: 默认消息文本未更改
+      AlreadyExists: 默认消息文本已存在
+      Invalid: 默认消息文本无效
+    PasswordComplexityPolicy:
+      NotFound: 未找到密码复杂性策略
+      Empty: 密码复杂性策略为空
+      NotExisting: 密码复杂性策略不存在
+      AlreadyExists: 密码复杂性策略已存在
+    PasswordLockoutPolicy:
+      NotFound: 密码锁定策略不存在
+      Empty: 密码锁定策略为空
+      NotExisting: 密码锁定策略不存在
+      AlreadyExists: 密码锁定策略已存在
+    PasswordAgePolicy:
+      NotFound: 密码过期策略不存在
+      Empty: 密码过期策略为空
+      NotExisting: 密码过期策略不存在
+      AlreadyExists: 密码过期策略已存在
+    OrgIAMPolicy:
+      Empty: 组织 IAM 策略为空
+      NotExisting: 组织 IAM 策略不存在
+      AlreadyExists: 组织 IAM 策略已存在
+  Project:
+    ProjectIDMissing: P缺少项目 ID
+    AlreadyExists: 项目以存在于组织中
+    OrgNotExisting: 组织不存在
+    UserNotExisting: 用户不存在
+    CouldNotGenerateClientSecret: 无法生成客户端秘钥
+    Invalid: 项目无效
+    NotActive: 项目不是启用状态
+    NotInactive: 项目不是停用状态
+    NotFound: 项目不存在
+    UserIDMissing: 缺少用户 ID
+    Member:
+      NotFound: 项目成员不存在
+      Invalid: 项目成员无效
+      AlreadyExists: 项目成员已存在
+      NotExisting: 项目成员不存在
+    MinimumOneRoleNeeded: 必须添加至少一个角色
+    Role:
+      AlreadyExists: 角色已存在
+      Invalid: 角色无效
+      NotExisting: 角色不存在
+    IDMissing: 丢失 ID
+    App:
+      AlreadyExists: 应用已存在
+      NotFound: 应用不存在
+      Invalid: 无效的应用
+      NotExisting: 应用不存在
+      NotActive: 应用不是启用状态
+      NotInactive: 应用不是停用状态
+      OIDCConfigInvalid: OIDC 配置无效
+      APIConfigInvalid: API 配置无效
+      IsNotOIDC: 应用不是 OIDC 类型
+      IsNotAPI: 应用不是 API 类型
+      OIDCAuthMethodNoSecret: 选择的 OIDC 身份验证方法不需要秘钥
+      APIAuthMethodNoSecret: 选择的 API 身份验证方法不需要秘钥
+      AuthMethodNoPrivateKeyJWT: 选择的身份验证方法不需要 Key
+      ClientSecretInvalid: Client Secret 无效
+    RequiredFieldsMissing: 缺少一些必填字段
+    Grant:
+      AlreadyExists: 项目授权已存在
+      NotFound: 授权不存在
+      Invalid: 项目授权无效
+      NotExisting: 项目授权不存在
+      HasNotExistingRole: 角色不存在与项目中
+      NotActive: 项目授权不是启用状态
+      NotInactive: 项目授权不是停用状态
+  IAM:
+    Member:
+      RolesNotChanged: 角色没有改变
+    MemberInvalid: 成员无效
+    MemberAlreadyExisting: 成员已存在
+    MemberNotExisting: 成员不存在
+    IDMissing: ID 丢失
+    IAMProjectIDMissing: IAM 项目 ID 缺失
+    IamProjectAlreadySet: IAM 项目 ID 已设置
+    IdpInvalid: IDP 配置无效
+    IdpNotExisting: IDP 配置不存在
+    OIDCConfigInvalid: OIDC IDP 配置无效
+    IdpIsNotOIDC: IDP 配置不是 OIDC 类型
+    LoginPolicyInvalid: 登录策略无效
+    LoginPolicyNotExisting: 登录策略不存在
+    IdpProviderInvalid: IDP 提供者无效
+    LoginPolicy:
+      NotFound: 默认登录策略不存在
+      NotChanged: 默认登录策略没有改变
+      NotExisting: 默认登录策略不存在
+      AlreadyExists: 默认登录策略已存在
+      RedirectURIInvalid: 默认重定向 URL 无效
+      MFA:
+        AlreadyExists: MFA 已存在
```

Comment:
In some local software, similar computer proper nouns are not translated, because users do not know what they are after translation, and it is difficult for maintenance personnel to accurately locate them. e.g. URL.

Reply:
thanks for clarification :)